modules/custom/az_core/az_core.install

```diff
@@ -12,3 +12,27 @@ function az_core_update_last_removed() {
   // Remove updates added before 2.6.0.
   return 920501;
 }
+
+/**
+ * Implements hook_requirements().
+ */
+function az_core_requirements($phase) {
+  $requirements = [];
+  if ($phase === 'runtime') {
+    $extension = 'az_barrio';
+    $library_name = 'arizona-bootstrap';
+    $library_info = \Drupal::service('library.discovery')->getLibraryByName($extension, $library_name);
+    if ($library_info['deprecated']) {
+      $message = $library_info['deprecated'];
+      $message = strtr($message, ['%library_id%' => $library_name]);
+      $requirements['az_barrio.deprecated_library'] = [
+        'title' => t('Deprecated library usage'),
```

Comment:
Nitpick: This isn't really detecting deprecated library usage, just the existence of deprecated libraries.

```suggestion
        'title' => t('Deprecated libraries'),
```